runtime/basilisk/src/xcm.rs

```diff
@@ -57,15 +57,17 @@ pub type LocalOriginToLocation = SignedToAccountId32<RuntimeOrigin, AccountId, R
 
 pub type Barrier = (
 	TakeWeightCredit,
-	AllowTopLevelPaidExecutionFrom<Everything>,
 	// Expected responses are OK.
 	AllowKnownQueryResponses<PolkadotXcm>,
-	// Subscriptions for version tracking are OK.
-	AllowSubscriptionsFrom<Everything>,
-	// Tinkernet Multisigs require DescendOrigin, this barrier makes sure it's followed by payment.
-	// The barrier will only pass when the origin is a Tinkernet multisig.
 	WithComputedOrigin<
-		AllowTopLevelPaidExecutionFrom<invarch_xcm_builder::TinkernetMultisigMultiLocation>,
+		(
+			AllowTopLevelPaidExecutionFrom<Everything>,
+			// Subscriptions for version tracking are OK.
+			AllowSubscriptionsFrom<Everything>,
+			// Tinkernet Multisigs require DescendOrigin, this barrier makes sure it's followed by payment.
+			// The barrier will only pass when the origin is a Tinkernet multisig.
+			AllowTopLevelPaidExecutionFrom<invarch_xcm_builder::TinkernetMultisigMultiLocation>,
```

Comment:
These two seem redundant?

Reply:
Not sure what exactly you mean. Removing `AllowTopLevelPaidExecutionFrom<invarch_xcm_builder...>` and moving `AllowSubscriptionsFrom` outside of `WithComputedOrigin`?

Reply:
so just the first of your comments

Reply:
removed `AllowTopLevelPaidExecutionFrom<invarch_xcm_builder::TinkernetMultisigMultiLocation>` from the barier.